fxpubsub/probe.go

```diff
@@ -0,0 +1,43 @@
+package fxpubsub
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/ankorstore/yokai/config"
+	"github.com/ankorstore/yokai/healthcheck"
+)
+
+type PubSubProbe struct {
+	config *config.Config
+	client *pubsub.Client
+}
+
+func NewPubSubProbe(config *config.Config, client *pubsub.Client) *PubSubProbe {
+	return &PubSubProbe{
+		config: config,
+		client: client,
+	}
+}
+
+func (p *PubSubProbe) Name() string {
+	return "pub-sub"
+}
+
+func (p *PubSubProbe) Check(ctx context.Context) *healthcheck.CheckerProbeResult {
+	topicName := p.config.GetString("modules.pubsub.topics.test")
```

Comment:
we better create a config key like `modules.pubsub.healthcheck.topics`, where users can list the topics they want to check